pkg/config/setup/config.go

```diff
@@ -1316,6 +1316,11 @@ func InitConfig(config pkgconfigmodel.Config) {
 	// TTL refresh period represents how frequently actively detected languages are refreshed by reporting them again to the language detection handler in the cluster agent
 	config.BindEnvAndSetDefault("language_detection.reporting.refresh_period", "20m")
 
+	// config for products implemented in the client libraries
```

Comment:
Can you move this block to be together with the rest of `admission_controller.auto_instrumentation.*` options?
https://github.com/DataDog/datadog-agent/pull/23618/files#diff-a7b6619528f155e64cefe12cc4ba753eb7688c87fdf9d96e1052a12eeb519df5R1085

Reply:
fixed in 675b22f098